src/transform/selecttransform.jl

```diff
@@ -1,25 +1,20 @@
 """
-    SelectTransform(dims::AbstractVector{Int})
+    SelectTransform(dims::Union{AbstractVector{Int}, AbstractVector{Symbol}})
```

Comment:
```suggestion
    SelectTransform(dims)
```
to reflect the updated constraints.